tools/cgeist/Test/Verification/invalid/basic.cpp

```diff
@@ -0,0 +1,3 @@
+// RUN: not cgeist %s --function=* -S
+
```

Comment:
change this to instead check for failure with ``// XFAIL: *``

Reply:
Done